### ISSUE TITLE
AP is always -1, using the pascal voc 2007 dataset

### ISSUE THREAD

**Issue by Zealoe** [smallcorgi/Faster-RCNN_TF#223]
I trained and then tested the model on the pascal voc 2007 dataset and got AP==-1. I did not change anything in the code. So what should I do ?

im_detect: 4950/4952 0.174s 0.002s
im_detect: 4951/4952 0.174s 0.002s
im_detect: 4952/4952 0.174s 0.002s
Evaluating detections
Writing aeroplane VOC results file
Writing bicycle VOC results file
Writing bird VOC results file
Writing boat VOC results file
Writing bottle VOC results file
Writing bus VOC results file
Writing car VOC results file
Writing cat VOC results file
Writing chair VOC results file
Writing cow VOC results file
Writing diningtable VOC results file
Writing dog VOC results file
Writing horse VOC results file
Writing motorbike VOC results file
Writing person VOC results file
Writing pottedplant VOC results file
Writing sheep VOC results file
Writing sofa VOC results file
Writing train VOC results file
Writing tvmonitor VOC results file
VOC07 metric? Yes
AP for aeroplane = -1.0000
AP for bicycle = -1.0000
AP for bird = -1.0000
AP for boat = -1.0000
AP for bottle = -1.0000
AP for bus = -1.0000
AP for car = -1.0000
AP for cat = -1.0000
AP for chair = -1.0000
AP for cow = -1.0000
AP for diningtable = -1.0000
AP for dog = -1.0000
AP for horse = -1.0000
AP for motorbike = -1.0000
AP for person = -1.0000
AP for pottedplant = -1.0000
AP for sheep = -1.0000
AP for sofa = -1.0000
AP for train = -1.0000
AP for tvmonitor = -1.0000
Mean AP = -1.0000
~~~~~~~~
Results:
-1.000
-1.000
-1.000
-1.000
-1.000
-1.000
-1.000
-1.000
-1.000
-1.000
-1.000
-1.000
-1.000
-1.000
-1.000
-1.000
-1.000
-1.000
-1.000
-1.000
-1.000
~~~~~~~~

**Comment by Zealoe** [comment]
I print the dets and the list is empty

**Comment by Zealoe** [comment]
I used lr==0.00001 and the loss has converged... I don't know the reason why it could not detect anything?
![image](https://user-images.githubusercontent.com/30797481/30642979-ac40662c-9e3f-11e7-9d1e-c40a05320437.png)

**Comment by indestinee** [comment]
@Zealoe I met the same problem, have fixed it?

**Comment by hxf930620** [comment]
i also met the same problem, have fixed it?

**Comment by indestinee** [comment]
@hxf930620 try use pre-trained model
Hope you can succeed :D

**Comment by hxf930620** [comment]
@indestinee   I changed the VGG16 network and the result turned to -1，do you know why?
